### PR TITLE
feat: add base dashboard layout

### PR DESCRIPTION
**Pull request by Extheoisah**

### CODE REVIEW

**Comment by Extheoisah** [comment]
Desktop screen:
image

Mobile screen:
1. 
image

2. 
image

**Comment by Extheoisah** [comment]
image

File: public/assets/icons/bell.tsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      {...props}
+    >
+      <path
+        d="M17.7936 14.4944C17.2733 13.5981 16.4999 11.0622 16.4999 7.75C16.4999 5.76088 15.7097 3.85322 14.3032 2.4467C12.8967 1.04018 10.989 0.25 8.99989 0.25C7.01077 0.25 5.10311 1.04018 3.69659 2.4467C2.29007 3.85322 1.49989 5.76088 1.49989 7.75C1.49989 11.0631 0.725515 13.5981 0.205202 14.4944C0.072331 14.7222 0.0018909 14.9811 0.000985404 15.2449C7.99065e-05 15.5086 0.0687412 15.768 0.200045 15.9967C0.331349 16.2255 0.520652 16.4156 0.748864 16.5478C0.977075 16.6801 1.23613 16.7498 1.49989 16.75H5.32583C5.49886 17.5967 5.95904 18.3577 6.62852 18.9042C7.29799 19.4507 8.13568 19.7492 8.99989 19.7492C9.8641 19.7492 10.7018 19.4507 11.3713 18.9042C12.0407 18.3577 12.5009 17.5967 12.674 16.75H16.4999C16.7636 16.7496 17.0225 16.6798 17.2506 16.5475C17.4787 16.4151 17.6678 16.225 17.799 15.9963C17.9302 15.7676 17.9988 15.5083 17.9979 15.2446C17.9969 14.9809 17.9265 14.7222 17.7936 14.4944ZM8.99989 18.25C8.53472 18.2499 8.08103 18.1055 7.70126 17.8369C7.3215 17.5683 7.03432 17.1886 6.87927 16.75H11.1205C10.9655 17.1886 10.6783 17.5683 10.2985 17.8369C9.91875 18.1055 9.46506 18.2499 8.99989 18.25Z"
+        fill="#7762FF"
```
Comment (daviroo) [inline public/assets/icons/bell.tsx:52]:
```suggestion
        fill={fill}
```

File: src/app/components/primitives/sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+          </Link>
+        </div>
+
+        <nav className="flex-1 items-center justify-center space-y-8">
+          {navItems.map((item) => (
+            <NavItem
+              key={item.href}
+              href={item.href}
+              icon={item.icon}
+              label={isOpen || window.innerWidth < 1024 ? item.label : ""}
```
Comment (daviroo) [inline src/app/components/primitives/sidebar.tsx:183]: We should probably move this `window.innerWidth` check out to a useEffect as you have done in the `ProfileDropdown` component.  The reason is to make it responsive to screen size changes after rendering.

File: public/assets/icons/overview.tsx [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        fill="#666666"
```
Comment (daviroo) [inline public/assets/icons/overview.tsx:49]:
```suggestion
        fill={fill}
```

**Review by daviroo — CHANGES_REQUESTED** [review]
Just a few small suggestions

File: src/app/components/primitives/sidebar.tsx (hunk 2 above)
Comment (Extheoisah) [reply src/app/components/primitives/sidebar.tsx:183]: Its actually not needed so I'll remove it

**Comment by Extheoisah** [comment]
> Just a few small suggestions

addressed https://github.com/hxckr-org/hxckr-web/pull/24/commits/8af4a5307fb26e1ec75dada9924419ee4988b32f

**Review by daviroo — APPROVED** [review]

**Pull request merged**